raphtory/src/core/tgraph.rs

```diff
@@ -312,6 +289,23 @@ impl TemporalGraph {
         self.layers[layer].add_edge_with_props(t, src_pid, dst_pid, props)
     }
 
+    pub(crate) fn delete_edge<T: InputVertex>(&mut self, t: i64, src: T, dst: T, layer: usize) {
+        let src_id = src.id();
+        let dst_id = dst.id();
+        // mark the times of the vertices at t
+        self.add_vertex(t, src)
+            .map_err(|err| println!("{:?}", err))
```

Comment:
println for error handling?

Reply:
let's fix that when we rewrite the graph :)